python/lsst/daf/butler/registries/oracleRegistry.py

```diff
@@ -22,9 +22,12 @@
 __all__ = ("OracleRegistry", )
 
 import re
+import __main__
```

Comment:
I think this import is sufficiently non-standard that it's worth adding a comment that you need this to be able to associate the connection with a particular executable script.